src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -87,6 +97,24 @@ export function attachToEventDelegator(eventDelegator: EventDelegator): void {
   });
 }
 
+function isSamePageWithHash(absoluteHref : string) : boolean {
+  const hashIndex = absoluteHref.indexOf('#');
+  // Excluding case hash="#"
```

Comment:
Could you extend the comment to say why we want to exclude this case? It's not obvious to me at least :)